web/src/js/__tests__/components/Command/CommandSpec.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import CommandBar from '../../../components/CommandBar'
+
+describe('CommandBar Component', () => {
+    let commandBar = renderer.create(
+            <CommandBar />),
+        tree = commandBar.toJSON()
+
+    it('should render correctly', () => {
+        expect(tree).toMatchSnapshot()
+    })
+})
```

Comment:
Let's please use react testing library and not `react-test-renderer` for all new tests. Here is a simple example:

https://github.com/mitmproxy/mitmproxy/blob/9b119c3dac3e57b29efd262edbb389e24a3d0214/web/src/js/__tests__/components/Header/MenuToggleSpec.js#L33-L43